x-pack/plugins/siem/public/components/source_destination/index.test.tsx

```diff
@@ -124,6 +125,25 @@ describe('SourceDestination', () => {
     ).toEqual('40B');
   });
 
+  test('it renders percent destination.bytes', () => {
+    const wrapper = mountWithIntl(<TestProviders>{getSourceDestinationInstance()}</TestProviders>);
```

Comment:
I think that TestProviders already gives you a `<I18nProvider>` and you can just use a regular `mount(`

Reply:
so you want me to change it everywhere